README.md

```diff
@@ -4,7 +4,18 @@ This is a collection of tutorials and examples to document, explain and illustra
 
 ## Getting started
 
-To get started basically follow these simple steps, which we will take you through in
+To get going with the tutorials you can either start directly in your browser or get
+ a local copy and experiment offline on your machine.
+ 
+ ### Quick start in current browser session
+ 
+ To start working in the notebooks directly in the browser, click 
+ [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/PTB-PSt1/PyDynamic_tutorials/introduce_mybinder).
+ 
+ ### Get a local copy to work offline
+
+To get started on your local machine basically follow these simple steps, which we
+will take you through in
 the following:
```

Comment:
I feel "To get started on your local machine, follow these simple steps:" is enough.

Reply:
Resolve in 3b9d25c93de4c3da941a6ac9a0f66722a2460769